app/views/online-booking/choose-date-calendar.html

```diff
@@ -0,0 +1,383 @@
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <form class="form" action="5reasonable-adjustments" method="post">
+
+
+        {%- from "govuk/components/tag/macro.njk" import govukTag -%}
+        {{ govukTag({
+          text: "In progress",
+          classes: "govuk-tag--purple"
+        }) }}
+        <h1 class="govuk-heading-l">
+          When do you want to visit?
+        </h1>
+
+        <p>Showing available times over the next 28 days</p>
+
+        <div id="calendar-slot-picker">
+          <h2 class="govuk-heading-m">
+            December 2023
+          </h2>
+
+          <ul class="govuk-list calendar-slots">
+            <li>M<span class="calendar-slots__day">on</span></li>
+            <li>T<span class="calendar-slots__day">ue</span></li>
+            <li>W<span class="calendar-slots__day">ed</span></li>
+            <li>T<span class="calendar-slots__day">hu</span></li>
+            <li>F<span class="calendar-slots__day">ri</span></li>
+            <li>S<span class="calendar-slots__day">at</span></li>
+            <li>S<span class="calendar-slots__day">un</span></li>
+          </ul>
+
+          <ul class="govuk-list calendar-slots">
+            <li></li>
+            <li></li>
+            <li></li>
+            <li><a href="#date-14-dec">14</a></li>
+            <li>15</li>
+            <li><a href="#date-16-dec">16</a></li>
+            <li><a href="#date-17-dec">17</a></li>
+          </ul>
+
+          <ul class="govuk-list calendar-slots">
+            <li>18</li>
+            <li>19</li>
+            <li><a href="#date-20-dec">20</a></li>
+            <li><a href="#date-21-dec">21</a></li>
+            <li>22</li>
+            <li><a href="#date-23-dec">23</a></li>
+            <li><a href="#date-24-dec">24</a></li>
+          </ul>
+
+          <ul class="govuk-list calendar-slots">
+            <li>25</li>
+            <li>26</li>
+            <li><a href="#date-27-dec">27</a></li>
+            <li><a href="#date-28-dec">28</a></li>
+            <li>29</li>
+            <li><a href="#date-30-dec">30</a></li>
+            <li><a href="#date-31-dec">31</a></li>
+          </ul>
+
+          <h2 class="govuk-heading-m">
+            January 2024
+          </h2>
+
+          <ul class="govuk-list calendar-slots">
+            <li>1</li>
+            <li>2</li>
+            <li><a href="#date-3-jan">3</a></li>
+            <li><a href="#date-4-jan">4</a></li>
+            <li>5</li>
+            <li><a href="#date-6-jan">6</a></li>
+            <li><a href="#date-7-jan">7</a></li>
+          </ul>
+
+          <ul class="govuk-list calendar-slots">
+            <li>8</li>
+            <li>9</li>
+            <li><a href="#date-10-jan">10</a></li>
+            <li><a href="#date-11-jan">11</a></li>
+            <li></li>
+            <li></li>
+            <li></li>
+          </ul>
+
+          <br>
+
+          <div class="day-slots" id="date-14-dec">
```

Comment:
Each of these `<div>` blocks represents the heading and set of radios at the bottom for a specific selected date. The `id` must match the relevant `href="#..."` mentioned above. The fields legend text is that day's full date.